data/common.yaml

```diff
@@ -20,6 +23,13 @@ mcollective::server_config:
   plugin.yaml: "/etc/puppetlabs/mcollective/generated-facts.yaml"
   registerinterval: 10
   registration: "Choria"
+  logfile: "/var/log/puppetlabs/mcollective.log"
```

Comment:
Adding the to the API is not a problem when the data is supplied by hiera from sane defaults - its a noop.

I'll have another look through this PR tomorrow and merge it, will rework how config is done to pull in libdir from `$mcollective::libdir` and ditto for the others rather than duplicating as we do now.